include/experimental/__p2642_bits/layout_padded.hpp

```diff
@@ -136,7 +137,7 @@ struct padded_extent {
       return {};
     }
     // Missing return statement warning from NVCC and ICC
-#if defined(__NVCC__) || defined(__INTEL_COMPILER)
+#if (defined(__NVCC__) || defined(__INTEL_COMPILER)) && !defined(__NVCOMPILER)
     return {};
 #endif
```

Comment:
All of these are contained in https://github.com/kokkos/mdspan/pull/356 and should disappear after rebasing.

Reply:
rebased